### PR TITLE
Improve LeftMenuHeader 

### PR DESCRIPTION
**Pull request by jasongitmail**
### Description of what you did:

This PR improves the LeftMenuHeader component and styles.

#### The current LeftMenuHeader has the following issues:
 1. It stretches the logo, making it soft.
 2. Doesn't allow a user to upload an image of a larger size--for example if a user has a logo that is short and wide, it will look bad in the current LeftMenuHeader because it will be much smaller than desired.
 3. Doesn't have a 2x retina version of the logo.

#### This PR improves it because: 
1. User can upload any size of logo, up to the width and height of the containing element.
2. Logo is never stretched & remains crisp.
3. User can provide 1x (non retina) and 2x (retina) images for crisp images on retina screens.
4. Logo filename is now `logo-admin.png`  and `logo-admin@2x.png`, instead of `logo-strapi.png`. This is an improvement because using a file named `logo-strapi.png` is misleading when a user adds their own logo, that isn't Strapi, with that name.

### Remaining todo for Strapi team please:
1. Rename the logo from `../../assets/images/logo-strapi.png` to `../../assets/images/logo-admin.png`
2. Add an identical logo but with twice the pixel dimensions at `../../assets/images/logo-admin@2x.png`

#### My PR is a:

- [x] 💥 Breaking change
- [ ] 🐛 Bug fix
- [x] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [x] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [x] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

**Comment by soupette** [comment]
Thanks for your PR, we will take care of producing the asset.

**Comment by klervilf** [comment]
Hi,

Wouldn't it be wiser to add an svg version of the logo ? Using a SVG, solves the problem of resolution no matter what computer you are using.
I guess it would be the most effective enhancement!
So here are both PNG files & a SVG.

[Logos_admin.zip]
(https://github.com/strapi/strapi/files/3680459/Logos_admin.zip)

**Comment by jasongitmail** [comment]
SVG is ideal. Thanks @klervilf  

@soupette Do you prefer:
1. SVG only (cleanest)
2. Or SVG with fallback to png and png@2x, as used in this commit, in case the user doesn't have an SVG version or their logo is too graphic for SVG (broader support).

I feel like the second might be easiest for the community since SVGs aren't easy to create for many people. I can update it for either.

**Comment by klervilf** [comment]
Hi,

I agree, I thing the wisest choice is to have a fallback to png. People might not always have SVG files, but it is important if they do that they'll be able to upload it. 

Thanks for your feedback !

**Comment by soupette** [comment]
@jasongitmail let's do the 2nd! thanks a lot

**Comment by jasongitmail** [comment]
Turns out Strapi's old CSS was shrinking the PNG logo from its actual pixel sizes and we actually needed smaller size PNGs. I generated new PNGs (108px and 216px widths) from the SVG. 

I committed `logo-admin.png`, `logo-admin@2x.png`, `logo-admin.svg` to the PR.

**Comment by jasongitmail** [comment]
@soupette Ran into a snag. I re-wrote the React component and SCSS for SVG with fallback to PNGs (see latest commits), and it works great _if the images exist_. But problem is that if any of the image files don't exist on disk, the webpack build fails with the error:
`Error: Module not found: Error: Can't resolve '../../assets/images/logo-admin.svg` (or similar if the PNGs are missing).

Any ideas how to make the image imports optional? (If not, we may have to stick with just SVG or just PNGs. The SVG and retina PNG look identical, so in such a case, I'd suggest sticking with PNGs because they're easier for most people to create.

File: packages/strapi-admin/admin/src/assets/images/logo-admin.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....32a1,1,0,0,1,.47-.64l21-12.07a1,1,0,0,1,1,0l15.64,9a1,1,0,0,1,.48.83l0,18.1a1,1,0,0,1-.48.83L35.58,43.52a1,1,0,0,1-.74.09l-6.3-1.78-5.9,3.43V55.71l3.45,2L50.16,43.78v-5.6l1.92-1.26v7.41a1,1,0,0,1-.48.83l-25,14.49a1,1,0,0,1-.48.13,1,1,0,0,1-.48-.13L21.19,57.1a1,1,0,0,1-.48-.83V44.71a1,1,0,0,1,.48-.83L27.92,40a1,1,0,0,1,.75-.09L35,41.65,54.87,30.23l0-17L40.23,4.76,20.13,16.33l-1.3,6.33a1,1,0,0,1-.46.64l-6.81,3.89a1,1,0,0,1-1,0L.54,21.39a1,1,0,0,1-.48-.83L0,15.47a1,1,0,0,1,.48-.84L25.54.13a1,1,0,0,1,1,0l6.16,3.74Z"/><path class="cls-1" d="M41.56,26.29a.94.94,0,0,1-.48-.13l-3.82-2.22a1,1,0,0,1-.48-.83v-4.4a1,1,0,0,1,.48-.84l3.82-2.2a1,1,0,0,1,1,0l3.83,2.19a1,1,0,0,1,.48.83v4.44a1,1,0,0,1-.48.83L42,26.16A.89.89,0,0,1,41.56,26.29Zm-2.85-3.74,2.86,1.67,2.85-1.65V19.25l-2.87-1.64L38.7,19.26Z"/><path class="cls-1" d="M11.55,33.74A1,1,0,0,1,11.07,32L14.42,30a1,1,0,0,1,1,1.66L12,33.61A.91.91,0,0,1,11.55,33.74Z"/><path class="cls-1" d="M10.84,39.57a1,1,0,0,1-.48-1.79l6.42-3.68a1,1,0,0,1,1.31.35,1,1,0,0,1-.35,1.32l-6.43,3.67A.9.9,0,0,1,10.84,39.57Z"/><path class="cls-1" d="M11.88,44.35a1,1,0,0,1-.84-.49,1,1,0,0,1,.37-1.31l7.75-4.35a1,1,0,1,1,.94,1.68l-7.75,4.34A.9.9,0,0,1,11.88,44.35Z"/></svg>
```
Comment (soupette) [inline packages/strapi-admin/admin/src/assets/images/logo-admin.svg]: Hi @jasongitmail thanks a lot for your contribution. However, I was wondering if you could make your PR against the develop branch where `node-sass` as been removed?

Thanks again!

**Comment by derrickmehaffy** [comment]
@jasongitmail any update to @soupette's request? Else we may need to close this PR until you are able to update it or if someone else wants to take over.

**Comment by jasongitmail** [comment]
Yes, closing. It would be helpful to have an answer to the blocker in my earlier question though https://github.com/strapi/strapi/pull/4113#issuecomment-538700197 for future contributors, or myself if I have time again, so it's possible to proceed.

**Pull request closed without merging**